src/mainboard/clevo/adl-p/ramstage.c

```diff
@@ -3,6 +3,56 @@
 #include <mainboard/gpio.h>
 #include <soc/ramstage.h>
 #include <smbios.h>
+#include <ec/system76/ec/commands.h>
+#include <drivers/efi/efivars.h>
+#include <fmap.h>
+#include <smmstore.h>
+
+#include <Uefi/UefiBaseType.h>
+
+#include <lib.h>
```

Comment:
Please order the includes in alphabetical order.  `Uefi/UefiBaseType.h` as a foreign include may be separated as is now.

Reply:
Alphabetized